compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -839,7 +840,8 @@ static void build_extern_assignment_function(Type* type)
 
   FnSymbol* fn = new FnSymbol("=");
   fn->addFlag(FLAG_ASSIGNOP);
-  fn->addFlag(FLAG_TRIVIAL_ASSIGNMENT);
+  // instead of marking fn with FLAG_TRIVIAL_ASSIGNMENT, mark the type as POD
```

Comment:
Since FLAG_TRIVIAL_ASSIGNMENT is gone, let's not mention it in the comment.
So the comment should probably be removed altogether as it does not add value.